tests/testthat/test-profile_emissions_upstream.R

```diff
@@ -338,7 +310,7 @@ test_that("outputs `profile_ranking_avg` at company level", {
   ecoinvent_europages <- read_csv(toy_ecoinvent_europages())
   isic_name <- read_csv(toy_isic_name())
 
-  out <- profile_emissions_upstream(
+  out <- profile_emissions_upstream_impl(
```

Comment:
SELF IMPORTANT can I revert this? Search for other cases of `_impl(` in tests and make sure they make sense.